### ISSUE TITLE
Blog - Write a post about the curriculum challenges and success

### ISSUE THREAD

**Issue by lauradereynal** [MozillaFoundation/digital-skills-observatory#147]
@chadsansing @secretrobotron  would you like to write a blog post with us about the first curriculum ? 

We can do it once we are back and have debriefed, but Im pasting this here so we don't forget.

**Comment by chadsansing** [comment]
Not sure I have anything of substance to contribute given others delivered/adapted the curriculum and observed participants responses, but would be a reader or help out however else I can. Maybe a facilitator would be a better co-author here?

**Comment by lauradereynal** [comment]
Yes no worries we would definitely help. I am thinking there might be something cool to write along these lines: 
- We had an hypothesis 
- Based on that hypothesis, we made this curriculum. 
- We faced some cross-cultural challenges, that are interesting to highlight. For example, it's difficult for Chad to think of an activity about Mpesa when he never had to use it before. 
- We learned even more how dramatically different our usage of the itnernet / smartphone is in differnet countries. 
- Which also impact how we think of digital skills. for example, there is a whole set of digital skills that are unique to here, and that might not be relevant in EU or USA. 
- This is important because it forces an organization like Mozilla to take into account some other cultures when making a web literacy map for example. (How do we take into account the fact that many of the poeple we are dealing with are illiterate ?)
- Based on all of these learnings, we can now create the next set of curriculum 
- We are thinking to put our attention towards XYZ XYZ XYZ 
- Here is where you can contribute. 

Maybe there is a framework around: 
- Personal Learnings 
- Project Learnings 
- Organizaitonal learnings

**Comment by chadsansing** [comment]
Don't forget to include what worked :)

Might also be useful to unpack the item mentioning the Web Literacy Map further. Are we talking about changing theory, delivery model, or both? Through Web Lit trainings in the United States, for example, @anmechung, @jvallera, @carot, @ldecoursy, others and I have come to think if digital skills as a separate (but related) precursor or complement to web literacy. (Right?)

It would be interesting to think about the distinctions and interrelations between the two in different cultures - would be of help in preparing our work, partnering with others, and helping others adapt our materials.

**Comment by lauradereynal** [comment]
Super. We have tones of notes on things that worked well :D

**Comment by secretrobotron** [comment]
I think the post @chadsansing wrote recently covers parts of this. We can follow up with more detail though. @LauraReynal what do you think?

**Comment by lauradereynal** [comment]
I agree that the existing blog post covers a lot of this. I also think it would be awesome to do some further reflections with Chad about the topics that we teach, and how they are received. 

Let's leave this ticket open as a reminder for now. 

@chadsansing would you be comfortable posting the blog post you wrote on the research blog? https://mozilla-foundation-research.herokuapp.com/ 
We can post it for you and link to your twitter if you would like.

**Comment by chadsansing** [comment]
@LauraReynal, cross-post away!

If we compose another post about the curriculum development, process, though, I would strongly encourage us to do it from the facilitators' POV. Perhaps you could survey or interview them and we could interweave their reflections with ours?

**Comment by lauradereynal** [comment]
@chadsansing 
exactly what I had in mind. Co-creating it with you, the community, the researchers. 
Maybe we can outline 3 questions that everyone reflects on ... or something like that.

**Comment by secretrobotron** [comment]
I still love this idea. Can write a skeleton and ask for collab.
